src/vs/platform/keybinding/common/keybindingLabel.ts

```diff
@@ -0,0 +1,27 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { keybindingLabelBackground, keybindingLabelForeground } from 'vs/platform/theme/common/colorRegistry';
+import { IColorTheme, ICssStyleCollector, registerThemingParticipant } from 'vs/platform/theme/common/themeService';
+
+registerThemingParticipant((theme: IColorTheme, collector: ICssStyleCollector) => {
```

Comment:
I would prefer this to live in `src/vs/workbench/browser/style.ts` where we have similar code already.

However: this widget is also being used in standalone editor:

![image](https://user-images.githubusercontent.com/900690/113697901-243ad280-96d4-11eb-8c48-3f9d86a68208.png)

So we need to ensure that a default is provided that works for the standalone editor. All our base widgets have this same problem.